docs/tasks/administer-cluster/kubeadm-upgrade-1-8.md

```diff
@@ -195,52 +202,74 @@ $ kubeadm upgrade apply v1.8.0
    find your CNI provider and see if there are additional upgrade steps
    necessary.
 
-## Upgrading your worker nodes
+## Upgrading your master and nodes
 
-For each worker node (referred to as `$WORKER` below) in your cluster, upgrade `kubelet` by executing the following commands:
+For each host (referred to as `$HOST` below) in your cluster, upgrade `kubelet` by executing the following commands:
+
+1. Prepare the host for maintenance, marking it unschedulable and evicting the workload:
+
+```shell
+$ kubectl cordon $HOST
+$ kubectl drain $HOST --ignore-daemonsets
+```
 
-1. Prepare the node for maintenance, marking it unschedulable and evicting the workload:
+When running `kubectl drain` against the master host, this error is expected and can be safely ignored:
 
 ```shell
-$ kubectl cordon $WORKER
-$ kubectl drain $WORKER
+node "master" already cordoned
+error: pods not managed by ReplicationController, ReplicaSet, Job, DaemonSet or StatefulSet (use --force to override): etcd-kubeadm, kube-apiserver-kubeadm, kube-controller-manager-kubeadm, kube-scheduler-kubeadm
 ```
 
-2. Upgrade the `kubelet` version on the `$WORKER` node by using a Linux distribution-specific package manager:
+2. Upgrade the Kubernetes package versions on the `$HOST` node by using a Linux distribution-specific package manager:
 
-If the node is running a Debian-based distro such as Ubuntu, run:
+If the host is running a Debian-based distro such as Ubuntu, run:
 
 ```shell
 $ apt-get update
-$ apt-get install -y kubelet
+$ apt-get upgrade
 ```
 
-If the node is running CentOS or the like, run:
+If the host is running CentOS or the like, run:
 
 ```shell
 $ yum update
-$ yum install -y kubelet
+$ yum upgrade
```

Comment:
Is this the right command? I'm not familiar with it, but thought `yum update` also upgrades packages...

Reply:
I wasn't quite sure. In my testing, it seems that `yum update` is all that's necessary, but I'd love for an expert to chime in. @timothysc, what do you think?

I've optimistically cut it down to just `yum update` for now.